docs/writing-rules/rule-syntax.md

```diff
@@ -416,7 +416,23 @@ rules:
       - pattern: module.$METHOD(...)
       - metavariable-regex:
           metavariable: $METHOD
-          regex: (^insecure$)
+          regex: (insecure$)
+    message: module using insecure method call
+    languages:
+      - python
+    severity: ERROR
+```
+
+This one matches all of the function calls in the example, returning a false positive on the `module.secure` call:
```

Comment:
```suggestion
The following example matches all of the function calls in the example, returning a false positive on the `module.secure` call:
```

Reply:
Good call -- I'm going to make a separate commit so I can rephrase the second instance of "example" at the same time.